core/local/steps/dispatch.js

```diff
@@ -19,11 +25,12 @@ let events, target, pouch, actions
 // Dispatch takes a buffer of AtomWatcherEvents batches, and calls Prep for
 // each event. It needs to fetch the old documents from pouchdb in some cases
 // to have all the data expected by prep/merge.
-module.exports = function (buffer /*: Buffer */, opts /*: { events: EventEmitter, prep: Prep, pouch: Pouch } */) {
+module.exports = function (buffer /*: Buffer */, opts /*: DispatchOptions */) /*: Buffer */ {
   events = opts.events
   target = opts.prep
   pouch = opts.pouch
-  buffer.asyncForEach(async (batch) => {
```

Comment:
I don't find the `Buffer` `asyncForEach` and `forEach` methods super straight forward and didn't want to remove them in case @nono had a specific use case in mind that I would see.
But I also thought about removing them since they aren't used anywhere (any more).

Reply:
It's OK for me to remove them if they are no longer used